content/develop/data-types/probabilistic/bloom-filter.md

```diff
@@ -10,18 +10,18 @@ categories:
 - kubernetes
 - clients
 description: Bloom filters are a probabilistic data structure that checks for presence
-  of an element in a set
+  of an item in a set
 linkTitle: Bloom filter
 stack: true
 title: Bloom filter
 weight: 10
 ---
 
-A Bloom filter is a probabilistic data structure in Redis Stack that enables you to check if an element is present in a set using a very small memory space of a fixed size.
+A Bloom filter is a probabilistic data structure in Redis Stack that enables you to check if an item is present in a set using a very small memory space of a fixed size.
 
-Instead of storing all of the elements in the set, Bloom Filters store only the elements' hashed representation, thus sacrificing some precision. The trade-off is that Bloom Filters are very space-efficient and fast.
+Instead of storing all the items in a set, a Bloom Filter stores only the items' hashed representation, thus sacrificing some precision. The trade-off is that Bloom Filters are very space-efficient and fast.
```

Comment:
```suggestion
Instead of storing all the items in a set, a Bloom Filter stores only the items' hashed representations, thus sacrificing some precision. The trade-off is that Bloom Filters are very space-efficient and fast.
```